src/main/kotlin/no/nav/pensjon/kalkulator/tech/security/egress/EnrichedAuthentication.kt

```diff
@@ -20,8 +20,8 @@ class EnrichedAuthentication(
     private val target: RepresentasjonTarget
 ) : Authentication {
 
-    fun getEgressAccessToken(service: EgressService): RawJwt =
-        egressTokenSuppliersByService.value[service]?.get() ?: RawJwt("")
+    fun getEgressAccessToken(service: EgressService, ingressToken: String?): RawJwt =
+        egressTokenSuppliersByService.value[service]?.apply(ingressToken) ?: RawJwt("")
```

Comment:
Tåler services å få token = `null`?

Reply:
`ingressToken` kreves bare for token-veksling (TokenX).
Dersom vi kaller en tjeneste med client credentials-token, er det ingen token-veksling, og da kan `ingressToken` være `null`.